apps/server/src/external/demo/app.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * This is a very minimal example for a websocket client
+ * You could use this as a starting point to creating your own interfaces
+ *
+ * This example does not handle disconnections
+ */
+/* eslint-disable */
+
+// this would need to resolve to the hostname of where ontime is running
+const socket = new WebSocket(`ws://${window.location.hostname}:4001/ws`);
```

Comment:
this will break if we are outside 4001, maybe it is OK but we should log something to the console?